specs/mgt-taxonomy-picker.md

```diff
@@ -0,0 +1,87 @@
+
+
+## Proposed Solution
+
+### Examples
+
+#### Show the first level children of the term set with the specified id.
+
+`<mgt-taxonomy-picker termset-id="138a652e-7f23-46f6-b480-13da2308c235"></mgt-taxonomy-picker>`
+
+#### Show the first level children of the term with the specified id under the term set with the specified id.
+
+`<mgt-taxonomy-picker termset-id="138a652e-7f23-46f6-b480-13da2308c235" term-id="a56caeb7-3b7d-4d22-93a9-0232e12905f6"></mgt-taxonomy-picker>`
+
+#### Show the first level children of the term with the specified id under the term set with the specified id in a different language.
+
+`<mgt-taxonomy-picker termset-id="138a652e-7f23-46f6-b480-13da2308c235" term-id="a56caeb7-3b7d-4d22-93a9-0232e12905f6" locale="fr-FR"></mgt-taxonomy-picker>`
+
+#### Cache the response from the API call for 50 seconds.
+
+`<mgt-taxonomy-picker termset-id="138a652e-7f23-46f6-b480-13da2308c235" cache-enabled="true" cache-invalidation-period="50000"></mgt-taxonomy-picker>`
+
+## Properties and Attributes
+
+| Attribute                 | Property                | Description                                                                                                                                                                                                                           | Type                       |
+| ------------------------- | ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------- |
+| placeholder                  | placeholder                 | The placeholder to use in the combobox. No default value is provided.                                                                                                                                        | string                     |
+| termset-id                  | termsetId                 | The Id of the termset where the terms are present. The terms under the termset will be shown if `term-id` is not passed                                                                                                                                         | string                     |
+| term-id                  | termId                 | The id of the term where the terms are present.                                                                                                                                            | Number                     |
+| locale                  | locale                | The locale of the terms that need to be displayed. This will be useful only when terms have multiple lables in different languages.                                                                                                                                                                        | String                     |
```

Comment:
```suggestion
| locale                  | locale                | The locale of the terms that need to be displayed. This will be useful only when terms have multiple labels in different languages.                                                                                                                                                                        | String                     |
```